integration-tests/common/gauntlet_common.go

```diff
@@ -4,25 +4,24 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"os"
-
 	"github.com/smartcontractkit/chainlink-starknet/integration-tests/utils"
+	"os"
 )
 
 var (
-	ethAddressGoerli = "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"
-	nAccount         string
+	strkAddressSepolia = "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d"
```

Comment:
is this only used on testnet?

Reply:
Can we group this and `chainIdTestnet` (+ maybe RPC) in a config struct somewhere? These constants are very brittle and split across several files

Reply:
Grouped under `integration-tests/config/config.go`